src/models/column.interface.ts

```diff
@@ -84,7 +85,7 @@ export interface Column<TData = any> {
   disableTooltip?: boolean;
 
   /** Any inline editor function that implements Editor for the cell value or ColumnEditor */
-  editor?: Editor | { model?: Editor; } | null;
+  editor?: Editor /*| { model?: Editor; }*/ | EditorConstructor | null;
```

Comment:
where were no references in the project to moel.Editor, so I thought this was a leftover

Reply:
no it's for my other libs usage only, so you can safely remove it from in here because like I said I no longer depend on this project for my own libs (like Slickgrid-Universal and others). So just remove the commented code please